### ISSUE TITLE
Requesting inventory error: TypeError: Cannot read properties of undefined (reading 'updateCharacteristic')

### ISSUE THREAD

**Issue by echamberlain** [grzegorz914/homebridge-enphase-envoy#121]
I'm getting the following error while trying to set up a new install:

```
[15/01/2024, 08:34:22] [homebridge-enphase-envoy] Device: envoy.local Envoy, Requesting inventory error: TypeError: Cannot read properties of undefined (reading 'updateCharacteristic'). Trying again in 15s.
[15/01/2024, 08:34:45] [homebridge-enphase-envoy] Device: envoy.local Envoy, Requesting inventory error: TypeError: Cannot read properties of undefined (reading 'updateCharacteristic'). Trying again in 15s.
```

Any ideas on how I can fix this?

**Comment by grzegorz914** [comment]
Go to,plugin settings Advanced Settings>>System>>Debug enable it and get the whole log.

**Comment by grzegorz914** [comment]
Fixed in 7.9.3

**Comment by echamberlain** [comment]
Here are the logs:

[homebridge-enphase-envoy.log](https://github.com/grzegorz914/homebridge-enphase-envoy/files/13941468/homebridge-enphase-envoy.log)

**Comment by grzegorz914** [comment]
This was other error, fixed in 7.9.4

**Comment by echamberlain** [comment]
Thanks for the quick turnaround.  The error is gone.
